scripts/core/superdesk-api.d.ts

```diff
@@ -2125,6 +2125,7 @@ declare module 'superdesk-api' {
         setValue: (value: IValue) => void;
         readOnly: boolean;
         config: IConfig;
+        template?: ITemplate;
```

Comment:
Is it used anywhere? It seems there's only a check if it's `null`, but actual value is never read.